colorization/src/tests.py

```diff
@@ -44,8 +44,10 @@ def test_step_preprocess_image(self):
         # test1: input a not existing file, should return FAILED
         img_path1 = os.path.join(os.path.abspath(os.path.dirname(__file__)),
                                  '../../Data/notexist.png')
-        result = proc.Preprocess(img_path1)
-        self.assertEqual(result, FAILED)
+        if not img_path1:
```

Comment:
why did you change this part?

I thought the idea here was to test, that the proc.Preproccess indeed fails